### DISCUSSION TITLE
"Section" Sound pack

### DISCUSSION THREAD

**Discussion by Switch-Spooks** [nzp-team/nzportable#288] [Other Content Releases]
Hi! I'm working on a soundpack for NZP! I originally started this because the 3DS sounds were kinda crunchy, but when I showed my friends the work, they wanted the pack too. I really hope you guys enjoy it! I'm still working on it, but this is MOSTLY finished!

-Uses SFX from different CoDs. (Mainly BO2 since that's what I have on hand, though some from BO3 and WaW)
-Adds David "Section" Mason as playable character (i just mixed voice lines from the BLOps 2 campaign into certain sounds)
-BLOps 2 menu SFX.

https://youtu.be/s7K-Bc6kl08

Warning! File size for the pack is 222MBs compressed! Make sure you have the space for the pack before you install to your system! Keep in mind this was made for the 3DS port, so I'm unsure as to how other systems will sound with it. The pack was generally kept to 22hz mono sound, 8 bit. Might end up lagging on O3DS hardware and PSP hardware.

https://www.mediafire.com/file/9j7f0l32nu9ur36/Section_Sound_Pack.zip/file

**Comment by Tadeoelbeo** [comment]
Where do you put character voice lines? And what do you name them?

**Reply by Switch-Spooks** [reply]
The voices are just mixed in with other SFX (like for instance, it's in the melee SFX). It's not a true character voice system.
